### ISSUE TITLE
Permission Denied Error

### ISSUE THREAD

**Issue by xianrusso** [oh-my-fish/oh-my-fish#729]
**Problem**

After re-installing `oh-my-fish` like so: `curl -L https://get.oh-my.fish | fish`

I receive the error: 
```
Updating https://github.com/oh-my-fish/packages-main master... 
fatal: could not create leading directories of '/home/xian/.cache/omf/https___github.com_oh-my-fish_packages-main_master': Permission denied
Error
```
![image](https://user-images.githubusercontent.com/22627739/68795053-323f0d80-061e-11ea-8850-d8a5b38e9a56.png)

However, when I run `fish`, the `omf` command is available for use.  The problem that keeps reoccurring is the inability to install any themes or plugins.  For example, when I enter the command `omf install lambda`, I receive this as the output:

![image](https://user-images.githubusercontent.com/22627739/68795298-ae395580-061e-11ea-925d-ce9349435a78.png)

The result of echoing for OMF_PATH and OMF_CONFIG, respectively are:
```
> echo $OMF_PATH
/home/USER/.local/share/omf
> echo $OMF_CONFIG
/home/USER/.config/omf 
```

These results confuse me since it seems as if the installation wants to create a directory in the /home/USER/.cache/omf/ directory, but there aren't any directories in the /.cache/ folder besides my Linuxbrew installation.

To make matters even more perplexing, when I try to access the fish_config file, I receive this error:

![image](https://user-images.githubusercontent.com/22627739/68795902-dd9c9200-061f-11ea-9a83-b629af901e92.png)

I checked which git was running, and it turns out that it's the git that I wanted to run: /usr/bin/git.  If someone could be so kind and explain to me in layman's terms as to why this git-related error keeps on persisting, I would be super grateful!

**Environment report**

after running `omf doctor`:

```
Oh My Fish version:   6
OS type:              Linux
Fish version:         fish, version 3.0.2
Git version:          git version 2.17.1
Git core.autocrlf:    no
Your shell is ready to swim.
```

For additional clarification, I'm running Ubuntu 18.04 using WSL on Windows 10.

**Comment by sagebind** [comment]
> If someone could be so kind and explain to me in layman's terms as to why this git-related error keeps on persisting, I would be super grateful!

Sure. In order to install packages, OMF must first download the [main package database](https://github.com/oh-my-fish/packages-main), which is a Git repository. The default path where this is stored is `~/.cache/omf/https___github.com_oh-my-fish_packages-main_master`. However, for some reason, Git does not have permission to `mkdir -p ~/.cache/omf`.

Why does Git not have permission to create `~/.cache/omf`? Unclear. Does your user own the `~/.cache` directory? Is `/usr/bin/git` secretly pointing to the Windows or cygwin version of Git? What happens if you run `mkdir -p ~/.cache/omf` manually?

**Comment by xianrusso** [comment]
@sagebind oh my word, you magnificent beast!—I solved it by changing ownership of that folder, i.e., `sudo chown (whoami) ~/.cache/`

Thank you for the quick reply and useful questions!
